Packs/CarbonBlackDefense/Integrations/CarbonBlackEndpointStandard/CarbonBlackEndpointStandard.yml

```diff
@@ -1773,28 +1770,28 @@ script:
     execution: false
     name: cbd-find-events-results
     outputs:
-    - contextPath: CarbonBlackDefense.Events.Results
-      description: The results of the event.
-      type: Unknown
-    - contextPath: CarbonBlackDefense.Events.Results.approximate_unaggregated
+    - contextPath: CarbonBlackDefense.Events.Results.job_id
+      description: The job_id of the events search.
+      type: String
+    - contextPath: CarbonBlackDefense.Events.Results.results.approximate_unaggregated
       description: The approximate number of unaggregated results.
       type: Number
-    - contextPath: CarbonBlackDefense.Events.Results.completed
+    - contextPath: CarbonBlackDefense.Events.Results.results.completed
       description: The number of completed results.
       type: Number
-    - contextPath: CarbonBlackDefense.Events.Results.contacted
+    - contextPath: CarbonBlackDefense.Events.Results.results.contacted
       description: The number of contacted results.
       type: Number
-    - contextPath: CarbonBlackDefense.Events.Results.num_aggregated
+    - contextPath: CarbonBlackDefense.Events.Results.results.num_aggregated
       description: The number of aggregated results.
       type: Number
-    - contextPath: CarbonBlackDefense.Events.Results.num_available
+    - contextPath: CarbonBlackDefense.Events.Results.results.num_available
       description: The number of events available in this search.
       type: Number
-    - contextPath: CarbonBlackDefense.Events.Results.num_found
+    - contextPath: CarbonBlackDefense.Events.Results.results.num_found
       description: The number of events found in this search.
       type: Number
-    - contextPath: CarbonBlackDefense.Events.Results.results
+    - contextPath: CarbonBlackDefense.Events.Results.results.results
```

Comment:
why so many results?

Reply:
to connect a job_id with his results, but it's really unnecessary I fixed it by pushing the job_id in the results